<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" hash="1159869897">
    <data-source source="LOCAL" name="Django default" uuid="d23bb335-84fe-474f-9a81-387e8bc7e032">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/RecetarioDjango/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\William\PycharmProjects\RecetarioDjango\db.sqlite3</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.ExplainPlanProvider.NullExplainPlanProvider</provider>
      </explain-plan-config>
      <libraries>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/xerial-sqlite-license.txt</url>
        </library>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/sqlite-jdbc-3.8.9.jar</url>
        </library>
      </libraries>
    </data-source>
  </component>
</project>